metricbeat/module/docker/_meta/config.yml

```diff
@@ -15,6 +15,9 @@
   # If set to true, replace dots in labels with `_`.
   #labels.dedot: false
 
+  # Docker module supports metrics collection from podman's docker compatible API. In case of podman set to true.
```

Comment:
```suggestion
  # Docker module supports metrics collection from podman's restful API. In case of podman set to true.
```

Reply:
Maybe this can be rephrased but my point is that we are using Podman's api which is compatible with docker. Because podman has also its own libpod api.

Reply:
Ah I see! Then write is like you said it above:
# Enables Podman Restful api which is compatible with docker. In case of podman set to true.